src/components/data-management/SamplesTable.jsx

```diff
@@ -90,24 +91,20 @@ const SamplesTable = forwardRef((props, ref) => {
         <SampleNameCell cellInfo={{ text, record, indx }} />
       ),
     },
-    ...fileUploadSpecifications[selectedTech]?.requiredFiles?.map((fileName, indx) => {
-      const fileNameWithoutExtension = fileName.key.split('.')[0];
-
-      return ({
-        index: 2 + indx,
-        title: <center>{fileName.displayedName}</center>,
-        key: fileNameWithoutExtension,
-        dataIndex: fileNameWithoutExtension,
-        width: 170,
-        onCell: () => ({ style: { margin: '0px', padding: '0px' } }),
-        render: (tableCellData) => tableCellData && (
-          <UploadCell
-            columnId={fileName.key}
-            sampleUuid={tableCellData.sampleUuid}
-          />
-        ),
-      });
-    }) || [],
+    ...fileUploadSpecifications[selectedTech]?.requiredFiles?.map((requiredFile, indx) => ({
```

Comment:
is it necessary / ok to have all those conditional operators? I know it's not introduced in this PR, I'm just asking if we are overusing the operator. Is it possible for `selectedTech` key not to exist in `fileUploadSpecifications` (sounds like it'd be a bug), or `requiredFiles` not existing

Reply:
I agree on the overuse of the conditional operators, I have been seeing that in a lot of parts of the platform. In some cases it's fine for the platform to break rather than have a `?` because the alternative is hiding bugs.

I'll check if it can be removed in this case btw

Reply:
In this case it seems it was added because `selectedTech` would be `null` on load (while the samples are loading). 

I switched these conditionals for an explicit if check on `selectedTech` not being `null` to make it more declarative and it seems to be working fine